src/api/entities/Asset/Metadata.ts

```diff
@@ -123,4 +123,24 @@ export class Metadata extends Namespace<Asset> {
 
     return new MetadataEntry({ ticker, type, id }, context);
   }
+
+  /**
+   * Gets the next local metadata ID for the Asset
+   */
+  public async getNextLocalId(): Promise<BigNumber> {
```

Comment:
Is it useful to be public? Do we expect users to need it?

Reply:
Users shouldn't need it but if we don't expose it as `public` then I won't be able to call it from where I am calling it.

I could inline it where its called, but it makes sense to me here. Otherwise we'd duplicate this. I am putting the `plus(1)` in this function to hide the "next" id is actually "latest" id

I could add a `@hidden` so it isn't visible in generated documentation.

Reply:
Yeah, I think adding `@hidden` is the best solution here.